### DISCUSSION TITLE
New Battery dedicated section

### DISCUSSION THREAD

**Discussion by andirsun** [elementary/switchboard-plug-power#187] [Ideas]
How about build a new section to see the battery stats in the power switchboard. 
We can get the history stats from the gnome-power-statistics and display in modern Gtk widgets 
This is the first step in this direction https://github.com/elementary/wingpanel-indicator-power/pull/190
From this : 
![Screenshot from 2021-07-01 20-47-12](https://user-images.githubusercontent.com/22928302/124208825-051ec780-daae-11eb-9729-65cfffda8263.png)
To something more modern like mac os does : 
![Screenshot from 2021-07-01 20-46-47](https://user-images.githubusercontent.com/22928302/124208863-17990100-daae-11eb-96b2-42e7d03fac17.png)

**Comment by cassidyjames** [comment]
This is being discussed at #156. Please use that issue for further discussion.
